### PR TITLE
css-positioning - Kristian Krog

### PR DESCRIPTION
**Pull request by krog-kristian**
relative-postioning
absolute-positioning
fixed-postioning

### CODE REVIEW

File: css-positioning/absolute.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  position: relative;
+}
+
+.meme > img {
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+  z-index: 1;
```
Comment (JenBowman1) [inline css-positioning/absolute.css:15]: z-index is not necessary here

**Review by JenBowman1 — CHANGES_REQUESTED** [review]
just one small change!

File: css-positioning/absolute.css (hunk 1 above)
Comment (krog-kristian) [reply css-positioning/absolute.css:15]: Haha, thank's tried that first and forgot to remove that one.

**Review by JenBowman1 — APPROVED** [review]
👍

**Pull request merged**